build.gradle.kts

```diff
@@ -179,14 +180,15 @@ project(":") {
     }
 
     val downloadClion = task<Download>("downloadClion") {
-        onlyIf { !file("${project.projectDir}/deps/clion-$clionVersion.tar.gz").exists() }
+        onlyIf { !file("${project.projectDir}/deps/$clionFullName.tar.gz").exists() }
```

Comment:
Do we really need rename all intermediate artifacts?
Maybe it's enough to rename only final folder?